plutus-core/untyped-plutus-core/test/Transform/Simplify.hs

```diff
@@ -227,11 +227,16 @@ multiApp = runQuote $ do
   pure app
 
 -- TODO Fix duplication with other golden tests, quite annoying
-goldenVsPretty :: (PrettyPlc a) => String -> String -> a -> TestTree
+goldenVsPretty ::
+  (PrettyBy (PrettyConfigReadable PrettyConfigName) a) =>
+  String ->
+  String ->
+  a ->
+  TestTree
 goldenVsPretty extn name value =
   goldenVsString name ("untyped-plutus-core/test/Transform/" ++ name ++ extn) $
-    pure . BSL.fromStrict . encodeUtf8 . render $
-      prettyPlcClassicDebug value
```

Comment:
You could just use `prettyPlcReadableDebug` here without defining a new data type. The `AsReadable` thing is useful for satisfying a `Show`, `Pretty`, `PrettyBy` etc constraint in a way that renders the thing readably regardless of the constraint.

Reply:
Oh that's convenient.. I didn't know that. I'll remove `AsReadableWithUniques` then.

Reply:
> I'll remove `AsReadableWithUniques` then.

Nah, keep it. It's definitely useful and I can see myself replacing some of `AsReadable` usages with `AsReadableWithUniques`.